docs/quickstart.md

```diff
@@ -267,9 +268,3 @@ $ kubectl get fdeploystatus echo-server -oyaml | yq .clusterStatus
 ```
```

Comment:
The `yq` command tool that `kubectl get fdeploystatus echo-server -oyaml | yq .clusterStatus` depends on in step 5 needs to be installed in advance. How about adding a prompt in this step?

Reply:
Not sure if we should require users to install `yq` just to filter out some lines... I think we can remove the ` | yq .clusterStatus` part and let users see the full output. @limhawjia WDYT?

Reply:
Fixed, @limhawjia @SOF3 @qclc PTAL